wagtail/contrib/redirects/templates/wagtailredirects/import_summary.html

```diff
@@ -0,0 +1,40 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+{% block titletag %}{% trans "Summary" %}{% endblock %}
+{% block content %}
+    {% trans "Summary" as summary_str %}
+    {% include "wagtailadmin/shared/header.html" with title=summary_str icon="redirect" %}
+    <div class="nice-padding">
+        <section id="summary">
+            <h2>{% trans "Summary" %}</h2>
+            <h3>{% blocktrans with total=import_summary.total successes=import_summary.successes errors=import_summary.errors_count %}Found {{ total }} redirects, created {{ successes }} and found {{ errors }} errors.{% endblocktrans %}</h3>
+
+            <a href="{% url 'wagtailredirects:index' %}" class="button">{% trans "Continue" %}</a>
```

Comment:
I think the continue button should show at the bottom, this is in line with many other final actions on a page in Wagtail.

Having continue last, after the errors, means it will hopefully be more likely for users to read the errors before clicking the 'continue' button.